alembic/env.py

```diff
@@ -41,6 +39,9 @@ def run_migrations_offline():
     script output.
 
     """
+    alembic_config = config.get_section(config.config_ini_section)
+    alembic_config['sqlalchemy.url'] = CONNECTION_STR
+
     url = config.get_main_option("sqlalchemy.url")
```

Comment:
Sorry I might not know about alembic a lot, but I think you just need to pass `CONNECTION_STR` to context.configure. Does this way work too? If yes, resolve that review comment.

Reply:
Seems fine, sorry